docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -49,7 +49,13 @@ notebooks:
         email: <String>
         role: <String>
         affiliation: <String>
-    
+
+    # A short text description of the notebook, 140 characters or less in length.
+    # Notebook-wide setting that will affect ALL branches/tags; only set this as needed in a main branch.
+    # If no topic has been specified, Dockstore will use the repository's "About" description as the topic.
```

Comment:
May need to check the code, but this may only work for GitHub (for example)

Reply:
Looks like it works for non-`.dockstore.yml` workflows and apptools, at least, see the `setTopicAutomatic` calls in `GitHubSourceCodeRepo`.  For notebooks and services, `.dockstore.yml` is the only registration method.

Reply:
I think that makes sense, mostly GitHub only was the main caveat I think

Reply:
> I think that makes sense, mostly GitHub only was the main caveat I think

The above comments are in the long example templates for `.dockstore.yml`, which is GitHub only, so it's implicit.  We can add verbiage about it being GitHub only, but not sure it's worth the extra words.  (IMHO, one of the issues with our docs is that the ratio of information to words is too high...)

Reply:
I don't know if that will always be the case (GitHub only), but fair enough